public/app/features/teams/TeamSettings.tsx

```diff
@@ -30,8 +30,12 @@ export const TeamSettings = ({ team, updateTeam }: Props) => {
   const [pendingRoles, setPendingRoles] = useState<Role[]>([]);
 
   const canUpdateRoles =
-    contextSrv.hasPermission(AccessControlAction.ActionUserRolesAdd) &&
-    contextSrv.hasPermission(AccessControlAction.ActionUserRolesRemove);
+    contextSrv.hasPermission(AccessControlAction.ActionTeamsRolesAdd) &&
+    contextSrv.hasPermission(AccessControlAction.ActionTeamsRolesRemove);
```

Comment:
The wrong checks were used here previously, we should check for permissions to change team roles not user roles.

Reply:
well spotted! ⚡